lib/include/hse_ikvdb/wal.h

```diff
@@ -15,6 +15,10 @@
 #define HSE_WAL_DUR_MS_DFLT        (100)
 #define HSE_WAL_DUR_MS_MAX         (1000)
 
+#define HSE_WAL_DUR_SZ_MIN         (32 * 1024u)
+#define HSE_WAL_DUR_SZ_DFLT        (8 * 1024 * 1024u)
+#define HSE_WAL_DUR_SZ_MAX         (128 * 1024 * 1024u)
```

Comment:
Is this an artificial limit, or does the code simply not work if size > 128 MiB?

Reply:
It's an artificial limit that I put.. If we go beyond 128 MiB it's highly likely that the durability interval (100ms) expires before we reach the dirty data threshold and that will trigger a sync.